Makefile

```diff
@@ -24,6 +24,10 @@ test-unit:
 build-tools:
 	$(GO_BUILD_ARGS) go build -o wni $(TOOLS_DIR)
 
+.PHONY: test-tools-e2e
+test-tools-e2e:
```

Comment:
Is `test-e2e-tools` better, this is more consistent with the other one.

Reply:
yes, use `make test-tools-e2e`

Reply:
but the other one use `test-e2e-prepared-node` naming. May I ask  why the order change?

Reply:
@Bowenislandsong please make the make target `test-e2e-tools`